EIPS/eip-4907.md

```diff
@@ -4,7 +4,7 @@ title: Rental NFT, ERC-721 User And Expires Extension
 description: Add a time-limited role with restricted permissions to ERC-721 tokens.
 author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>
 discussions-to: https://ethereum-magicians.org/t/idea-erc-721-user-and-expires-extension/8572
-status: Review
+status: Last Call
```

Comment:
```suggestion
status: Last Call
last-call-deadline: YYYY-MM-DD
```
This needs a last call deadline.  It should be *at least* 14 days in the future.  Due to editor response times being a bit slow for ERCs, I recommend setting it 3-4 weeks in the future so you don't run into problems.